pom.xml

```diff
@@ -48,7 +48,7 @@
     <properties>
         <jenkins.version>1.642.3</jenkins.version>
         <java.level>7</java.level>
-        <workflow.version>1.14</workflow.version>
+        <workflow.version>2.0</workflow.version>
```

Comment:
:ant: this [does not make any sense](https://jenkins.io/blog/2016/04/22/pipeline-2/); specify the version of each plugin separately